src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowExtended.groovy

```diff
@@ -502,6 +528,41 @@ class FlowExtended {
         return new FlowBidirectionalExam(forward, reverse)
     }
 
+
+    LldpData sendLldpFromTgToEndpoint(TraffExamService tgService, FlowEndpointV2 endpoint, LldpData lldpData,
```

Comment:
I guess these (this and below ones) methods shouldn't be in FlowExtended class. May be TraffExamService would be a good place? Or a new class like `ConnectedDeviceTrafficGenerator`?

Reply:
updated.